### ISSUE TITLE
Convert Z-Values excel logic into a C++ program

### ISSUE THREAD

**Issue by lepatrick714** [lepatrick714/Full_Sensor_Senior_Design#1]
Excel sheet is located here: https://docs.google.com/a/ucr.edu/spreadsheets/d/1BaylT-DU5b9FI94PveNpNFw9sW1sw6nB-0iwIZY09AA/edit?usp=sharing

The input values are highlighted in yellow. 
The output value is highlighted in red. 

This program should be able to take in 6 inputs, compute the result ( should be a scaler ), and output the result to a file. 

Please ensure that the program is as optimal as possible.

w & BO are not constants, Everything else are constants.

**Comment by richie148860** [comment]
Just a note this program should be able to take 8 inputs:
pressure, temperature, and methane, ethane, propane, CO2, butane, and nitrogen mole fractions

**Comment by lepatrick714** [comment]
This is correct. @richie148860 Could you write up the equations in a separate sheet? It would be easier for us to fully understand how to implement it. 

Cheers!

**Comment by lepatrick714** [comment]
@richie148860 Could you take a look at issue, 

"Porting the codebase to github and refactoring of parts of the codebase" 

There are some undefined code.

**Comment by lepatrick714** [comment]
I'll be moving the constants to a separate file

**Comment by lepatrick714** [comment]
Here's an initial pull request: 

https://github.com/lepatrick714/Full_Sensor_Senior_Design/pull/16
